mail_activity_reply_creator/models/mail_activity.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 ForgeFlow S.L.
```

Comment:
This file is new, does it come from ForgeFlow since 2018?

Reply:
sorry i've never took care of copyright )